Lib/fontbakery/specifications/name.py

```diff
@@ -406,6 +406,34 @@ def com_adobe_fonts_check_postscript_name_cff_vs_name(ttFont):
     yield PASS, ("Name table PostScript name matches CFF table FontName.")
 
 
+@check(
+  id='com.adobe.fonts/check/name_id_6_consistency',
+  conditions=['not is_cff'],  # e.g. TTF or CFF2
+  rationale="""
+  The PostScript name entries in the font's 'name' table should be
+  consistent across platforms.
+
+  This is the TTF/CFF2 equivalent of the CFF 'postscript_name_cff_vs_name'
+  check above.
```

Comment:
I would drop the "above" from this sentence. It makes sense for one reading the code, but not for a user reading the medatada in some other context in the future (we still do not expose the metadata, but that's the plan).

Reply:
OK. Done. Thanks.